### PR TITLE
Feature/select favorites

### PR DESCRIPTION
**Pull request by marialitwa**

### CODE REVIEW

File: pages/favorites.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                <AnswerCard
+                  answerText={[answerText]}
+                  toggleFavorite={toggleFavorite}
+                />
+              </div>
+            );
+          } else {
+            return null;
+          }
+        })}
```
Comment (JessicaLoers) [inline pages/favorites.js:46]: **I would like to show you two other ways for conditional rendering, the first is my favorite 🙂** 

1. [Inline If with Logical && Operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)
 ```suggestion
      {morningAnswers?.map(
          (answerText) =>
            answerText.isFavorite && (
              <div>
                <StyledEntryDate>{stringToLocaleDate(answerText.createdDate)}. 
               </StyledEntryDate>
                <AnswerCard answerText={[answerText]} toggleFavorite={toggleFavorite} />
              </div>
            )
        )}
```

File: pages/favorites.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
```
Comment (JessicaLoers) [inline pages/favorites.js:63]: **2. [Inline If-Else/ternary operator with Conditional Operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator)**


```suggestion
          {morningAnswers?.map((answerText) =>
          answerText.isFavorite ? (
            <div>
              <StyledEntryDate>{stringToLocaleDate(answerText.createdDate)}</StyledEntryDate>
              <AnswerCard answerText={[answerText]} toggleFavorite={toggleFavorite} />
            </div>
          ) : null
        )}
```

**Review by JessicaLoers — APPROVED** [review]
**Hello @marialitwa** 
**Your code is very clean and well structured.**  🚀

You could take another look at the semantics of your elements. There are quite a few paragraphs and I'm sure you can find more semantically descriptive elements.

Furthermore, I have left you two examples for a conditional rendering as a suggestion. 
Your way isn't wrong, but maybe you like the `Logical && Operator-Method `for conditional rendering as much as I do 😬

And please add the aria-label to the button with nested svg. 

If you adjust the changes, your code is approved - I trust you 😉

**Comment by marialitwa** [comment]
Dear Jessica, thanks for your feedback! Always learning from it and being reminded of stuff I forgot about due to the loads of input 😁 Thank you! I refactored the code as suggested 🙂

**Pull request merged**